interactive_setup.sh

```diff
@@ -93,6 +92,25 @@ fi
 replicationBackend=$(docker ps | grep -c "\-$org-replication-backend")
 aamBackendService=$(docker ps | grep -c "\-$org-aam-backend-service")
 
+if [ "$app" == 0 ]; then
+  if [ -n "$2" ]; then
+    baseConfig="$2"
+  else
+    echo "Which basic config do you want to include? (e.g. [default], basic, codo, ...)"
+    read -r baseConfig
+
+    if [ ! "$baseConfig" ]; then
+      baseConfig=default
```

Comment:
should we rename baseConfigs/basic to baseConfigs/default then?

Reply:
Sometimes we may need an empty setup? Basic already bring some configuration...

Reply:
"empty" doesn't start, though. It will endlessly "load config" because the config doc is not available. 

Maybe we do want a "default" setup though, where the config is (almost) empty but present. That's a good point.

Reply:
Yes, good point. Then we should add the absolut minimum config for a application start as "default".

Reply:
added